README.md

```diff
@@ -62,6 +62,29 @@ The process is different for each provider, so you may refer to their documentat
 
 ![Screenshot showing how to use the chat](./img/3-usage.png)
 
+## Using ChromeAI
+
+> [!WARNING]
+> Support for ChromeAI is still experimental and only available in Google Chrome.
```

Comment:
Do you know if there a way to detect the browser, and have this provider available for chrome only ?

Reply:
Maybe just checking for `window.ai` availability could be enough?

Reply:
Since we specify the name of the model as part of the settings definition:

```json
"enum": ["None", "Anthropic", "ChromeAI", "MistralAI"]
```

Not sure there could be a nice way to dynamically enable / disable support for ChromeAI, based on `window.ai`? Unless we manually check for "ChromeAI" and remove it from the settings if `window.ai === undefined`.

Reply:
> Not sure there could be a nice way to dynamically enable / disable support for ChromeAI, based on `window.ai`?

Yes, that settings management is already hazardous, don't no neither if we want to add more manual changes.
Maybe the way to go will be to add a dedicated configuration panel that doesn't rely on settings...

Reply:
> Maybe the way to go will be to add a dedicated configuration panel that doesn't rely on settings...

Agreed, sounds like it would give more flexibility 👍

Reply:
I opened https://github.com/jupyterlite/ai/issues/35